src/pieces/assets/assets_identifiers_ws.py

```diff
@@ -3,31 +3,33 @@
 import websocket
 from pieces.settings import Settings
 import threading
+from base_websocket import BaseWebsocket
+import json
+from typing import Dict ,Any
 
 
-class AssetsIdentifiersWS:
-    def __new__(cls,*args,**kwargs):
+class AssetsIdentifiersWS(BaseWebsocket):
+    def __new__(cls, *args, **kwargs):
         if not hasattr(cls, 'instance'):
             cls.instance = super(AssetsIdentifiersWS, cls).__new__(cls)
         return cls.instance
-    
+
     def __init__(self, on_message_callback):
```

Comment:
Can we remove the init?
We should start the ws in the cli loop so we can change the cli loop file to do BaseWebsocket.start_all

Reply:
addressed in [efad189](https://github.com/pieces-app/cli-agent/pull/167/commits/efad1899ee832844715214fb6d953f4e60315851) commit

Reply:
Also @Bishoy-at-pieces , if we are to use the start_all from the BaseWebsocket eventually , should we also remove the open_websocket and the close_websocket_connection also , as they would then become redundant to the start_all and close_all respectively ?

Reply:
I would like to leave them 
Later we might need to connect to specific websockets not all for now we might need to start all

Reply:
okay.